### DISCUSSION TITLE
will Learning Equality participate in GSOC 2024?

### DISCUSSION THREAD

**Discussion by AnkitMishra007** [learningequality/kolibri#11714] [Q&A]
will Learning Equality participate in GSOC 2024?

**Comment by MisRob** [comment]
Hello @AnkitMishra007, we're planning for it, however we need to wait until we're confirmed as the mentoring organization for GSOC 2024. @radinamatic may know more.

**Comment by Rishabh-1126** [comment]
@radinamatic will you participate this year in gsoc?

**Reply by radinamatic** [reply]
We are going to apply, and Google will announce the accepted mentoring organizations on February 21, 2024.
Be sure to check the https://summerofcode.withgoogle.com/ for any updates.

**Comment by AnkitMishra007** [comment]
OK
